src/components/productsList/_productsList.scss

```diff
@@ -0,0 +1,121 @@
+$media-tablet: 768px;
+$media-desktop: 1024px;
+$padding-mobile: 20px;
+$padding-desktop: 40px;
+
+.products-list {
+  display: flex;
+  flex-direction: column;
+  margin: 40px 10px 10px;
+
+  * {
+    box-sizing: border-box;
+  }
+
+  @media screen and (min-width: $media-desktop) {
+    flex-direction: row;
+  }
+}
+
+.products-list-nav {
+  $self: &;
+  background: $primary;
+  position: relative;
+  padding: $padding-mobile;
+  border-radius: 8px;
+
+  @media screen and (min-width: $media-tablet) {
+    padding: $padding-desktop;
+  }
+
+  @media screen and (min-width: $media-desktop) {
+    flex: 1 0 50%;
+    padding: calc(#{$padding-mobile} + 6%) $padding-desktop;
+  }
+  
+  &__title {
+    color: $secondary;
+    font-size: 16px;
+    font-weight: normal;
+    margin: 0;
+
+    @media screen and (min-width: $media-tablet) {
+      display: none;
+    }
+  }
+
+  &__items {
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    transform: translateY(100%);
+    width: 100%;
+    background: $secondary;
+    padding: $padding-mobile;
+    display: none;
+
+    #{$self}--opened & {
+      display: block;
+    }
+
+    @media screen and (min-width: $media-tablet) {
+      display: block;
+      position: relative;
+      transform: translateY(0);
+      background: transparent;
+      padding: 0;
+    }
+
+    li {
+      margin-bottom: 30px;
+
+      &:last-of-type {
+        margin-bottom: 0;
+      }
+    }
+  }
+
+  &__link {
+    color: $dark;
+    text-decoration: none;
+    display: block;
+
+    @media screen and (min-width: $media-tablet) {
+      color: $secondary;
+      font-size: 24px;
+    }
+
+    &--active {
+      color: $primary;
+
+      @media screen and (min-width: $media-tablet) {
+        color: $secondary;
+        text-decoration: underline;
+      }
+    }
+  }
+}
+
+.products-list-content {
```

Comment:
This could be part of the above selector with `&-content` and should follow the BEM methodology (`product-list__content`)